runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowRunner.java

```diff
@@ -197,6 +197,10 @@
 })
 public class DataflowRunner extends PipelineRunner<DataflowPipelineJob> {
 
+  /** Experiment to "disable unbounded pcollection checks turning batch into streaming". */
+  public static final String DISABLE_UNBOUNDED_PCOLLECTION_CHECKS_TURNING_BATCH_INTO_STREAMING =
+      "disable_unbounded_pcollection_checks_turning_batch_into_streaming";
```

Comment:
"unsafely_attempt_to_process_unbounded_data_in_batch_mode"

Reply:
Done.

Reply:
(Btw my bad, totally missed this in the previous comment due to the Kafka separation.)

Reply:
Of course I couldn't do it from the GitHub UI. NOW done. (https://github.com/apache/beam/pull/16773/commits/80a19be50ce3121faa2e7386ae24175bd720363d)